internal/resourcemodifiers/resource_modifiers_test.go

```diff
@@ -189,7 +341,7 @@ func TestGetResourceModifiersFromConfig(t *testing.T) {
 				return
 			}
 			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("GetResourceModifiersFromConfig() = %v, want %v", got, tt.want)
+				t.Errorf("GetResourceModifiersFromConfig() = %#v, want %#v", got, tt.want)
```

Comment:
I don't think we should need # here...
```suggestion
				t.Errorf("GetResourceModifiersFromConfig() = %#v, want %#v", got, tt.want)
```